crates/router/src/macros.rs

```diff
@@ -1 +1,11 @@
 pub use common_utils::{collect_missing_value_keys, newtype};
+
+#[macro_export]
+macro_rules! get_formatted_date_time {
+    ($date_format:tt) => {{
+        let format = time::macros::format_description!($date_format);
+        time::OffsetDateTime::now_utc()
+            .format(&format)
+            .change_context(errors::ConnectorError::InvalidDateFormat)
```

Comment:
Would have preferred using the fully qualified name for this error usage: `router::core::errors::ConnectorError`.